client/src/bundles/vendor.js

```diff
@@ -51,8 +51,6 @@ require('../../../thirdparty/jquery-entwine/dist/jquery.entwine-dist.js');
 require('../../../thirdparty/jquery-cookie/jquery.cookie.js');
 require('../../../thirdparty/jquery-query/jquery.query.js');
 require('../../../thirdparty/jquery-form/jquery.form.js');
-require('../../../thirdparty/jquery-notice/jquery.notice.js');
```

Comment:
We no longer need this plugin, because it fully got reimplemented in react. There's a polyfill to make sure existing calls still works.